docs/guide.mdx

```diff
@@ -0,0 +1,398 @@
+---
+sidebar_position: 10
+sidebar_label: "Walkthrough"
+id: deployment-guide
+title: Deployment Guide
+description: Deployment Guide
+slug: /walkthrough
+---
+
+This guide provides detailed instructions for deploying a Kwil network and for interacting with the nodes within the network.
+
+## Deploy A Standalone Kwil Node
+
+### Configuration
+
+Initialize the root directory with the required configuration files by running the below command.
+
+```bash
+$ kwil-admin setup init -o ~/.kwild
+Successfully initialized node directory: ~/.kwild
+```
+
+This generates node's private key, genesis and config files in the specified root directory.
+```bash
+$ tree ~/.kwild
+~/.kwild
+├── abci
+│   ├── config
+│   │   └── genesis.json
+│   └── data
+├── config.toml
+└── private_key
+
+4 directories, 3 files
+```
+The configuration is initially set to its default values. These can be updated manually or overridden using command line flags or environment variables. Refer to the [configuration guide](./daemon/kwild#configuration-override) for more information on this. 
+
+Notable options to configure are:
+#### Genesis File
+The Kwil blockchain currently does not support gas costs, so ensure that `without_gas_costs` set to `True` in the genesis file. 
```

Comment:
I think kwilteam/kwil-db#324 has by default set to `true`

Reply:
It does. I still thought it's worth mentioning.